curp/src/server/gc.rs

```diff
@@ -1,97 +1,152 @@
-use std::{sync::Arc, time::Duration};
+use std::time::Duration;
 
-use parking_lot::Mutex;
-use tokio::time::Instant;
+use crate::{cmd::Command, server::cmd_board::CmdBoardRef};
 
-use super::spec_pool::SpeculativePool;
-use crate::{cmd::Command, server::cmd_board::CommandBoard};
-
-/// How often spec should GC
-const SPEC_GC_INTERVAL: Duration = Duration::from_secs(10);
+use super::spec_pool::SpecPoolRef;
 
 /// How often cmd board should
```

Comment:
Please modify the comment message in that the `GC_INTERVAL` is used in both `gc_spec_pool` and `gc_cmd_board`, but the comment only about `cmd_board`

Reply:
Fixed.